variables.mk

```diff
@@ -9,7 +9,7 @@ CONTROLLER_GEN_BIN=./bin/controller-gen
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 
 # image vars
-BASE_IMAGE ?= golang:1.17
+BASE_IMAGE ?= golang:1.21
```

Comment:
change this is not enough. as in our internal csi build,  we override the BASE_IMAGE  in ci build.  you need also change the ci build jenkins file.

Reply:
I've changed BASE_IMAGE in ci build

Reply:
ok, but I did not see the jenkins ci file change PR in our internal github.